packages/documentation/docs/controls/_blind_styleguide.md

```diff
@@ -10,76 +10,58 @@ import SourceAngularHtml from './../auto-generated/previews/angular/blind.html.m
 
 import Playground from '@site/src/components/Playground'
 
-Blinds are UI controls that allow the users to hide or reveal content by clicking on a control element.
+Blinds are UI controls that allow the users to hide or reveal content by clicking on a control element. Blinds can display a large amount of content in a compact space or present information in an organized and hierarchical way. Blinds reduce the user's cognitive load by removing clutter and less important information from an interface. We typically don't use blinds if the content is central to the user's task due to its reduced visibility and accessibility.
 
-![Blind overview](/img/pattern_illustrations/Blind_overview.png)
+Blinds consist of a header section on the top and a content section below. The header section contains a chevron on the left followed by the blind's label. Within the content section, content can be placed freely.
 
-Go to:
+![Blind overview](https://www.figma.com/file/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=2%3A2&mode=design&t=9faEnH99BaAxqCGM-1)
 
-- [States](#states)
-- [Dos and don't](#dos-and-donts)
+1. Header section
+2. Content section
 
-## Usage
+## Types
 
-Blinds are often used to display a large amount of content in a compact space, or to present information in an organized and hierarchical way.​ Blinds can reduce cognitive load of the user be removing clutter and less important information from an interface.
+Multiple blinds can be placed below each other to create an accordion. The recommended distance between the blinds is `0.5rem`. Blinds within an accordion can either close and open independently or dependently depending on the design requirements.
 
-### When to use
+![Accordion](https://www.figma.com/file/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=2%3A655&mode=design&t=9faEnH99BaAxqCGM-1)
 
-Use blinds when you want to display a large amount of related content, and the content can be organized into discrete sections. This allows users to reveal only the content that they are interested in, and helps to reduce clutter and improve readability.​
+## Variants
+Multiple blind variants are available: 
+- **Filled**: Default variant
+- **Outline**: Variant for lower visual emphasis
+- **Primary**: Variant for high visual emphasis
+- **State-related variants**: Alarm, critical, warning, success, info, neutral
 
-### When not to use
 
-Consider the context in which the blinds will be used. If the content is central to the user's task, using a blind might reduce the content's visibility and accessibility.
+![Blind variants](https://www.figma.com/file/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=929%3A47485&mode=design&t=9faEnH99BaAxqCGM-1)
```

Comment:
Illustration on figma shows no differnet variants:
![image](https://github.com/siemens/ix/assets/2144313/6de3c809-3cf2-4b22-8b56-683720024c12)

Reply:
Good catch, it's fixed now